script/Ciel/Day 14/14_01B_03_CIEL14B_4_1B.txt

```diff
@@ -156,12 +166,16 @@ In her hands, one of the two people who were allowed to remain this world is cap
 [sha:4fb6f0a633f6269afc42e601f8e8fbcd858806db]{
 -- Page 29, Offset 40331.
 -- 嗜虐の笑みを浮かべた後、指先で命の感触を確かめる。
-After raising a sadistic smile, she checks the texture of the life with her fingertips.
+Smiling sadistically,
+ she assures herself of the sensation of life between her fingertips.
 }
 [sha:1329ac193ef6cf349f829a01bb324c035744cb43]{
 -- Page 29, Offset 40332.
 -- 足元の世界から<知|き><覚|く>に堪えない、ひどい罵詈雑言が届いたが、そんなものは歯牙にかけるまでもない。
-She received terrible insults from the world at her feet that she couldn't bear to hear, but it wasn't enough to warrant her biting back.
+// Assuming this is Shiki yelling profanity?
+From the world at her feet,
+ she receives a stream of abuse that she can hardly bear to hear,
+ but she takes no notice of it.
 }
 [sha:fba2a9a397d6f5dc18720bf07557b301658af6cd]{
 -- Page 30, Offset 40333. Mods: @n.
```

Comment:
Ohhh, the second here is a bit bad, but it's a hard line.
Maybe something really drastic?
>Right, my precious little doll?

>You, at least, will always return to normal no matter how many times I crush you.